src/java.base/share/classes/java/lang/SecurityManager.java

```diff
@@ -127,8 +125,7 @@
  * <tr>
  *   <th scope="row">null</th>
  *   <td>None</td>
- *   <td>Success or throws {@code SecurityException} if not permitted by
- * the currently installed security manager</td>
+ *   <td>Always throws {@code UnsupportedOperationException}</td>
```

Comment:
Not sure "Always" is needed, could just be "Throws UOE"

Reply:
This is the same as the existing words for "disallow".

Reply:
I think I agree with Lance that "Throws UOE" would be clearer in this table.

Reply:
Suggestion accepted.